### PR TITLE
LPS-179056 | New Test automation WithSpecialCharacters

### PR DESCRIPTION
**Pull request by andreluis2022**
Description:
Create new test Test to create a new dataset (name with special characters)

Plan:
- Given access to Datasets admin page// App Menu > Control Panel > Datasets
- When The user goes to add Datasets // clicking on the New dataset plus button
- And When you choose the Provider called channel
- And When clicking on the Save button
- And Then Confirm that the dataset is created // Assert that name and provided as expected

JIRA Ticket:
https://issues.liferay.com/browse/LPS-179056

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase [hunk 1]
```diff
@@ -92,4 +92,32 @@ definition {
 
 	}
 
+	@description = "LPS-179056 - Confirm that the user can create a data set view"
+	@priority = 5
+	test WithSpecialCharacters {
+		property custom.properties = "feature.flag.LPS-164563=true";
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase:98]: please remove this property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase [hunk 2]
```diff
@@ -92,4 +92,32 @@ definition {
...
+			PortletEntry.inputName(name = "~!@#$%^&*(){}[],.<>/?");
+		}
+
+		task ("And When you choose the Provider called channel ") {
+			FrontendDataSetAdmin.selectProvider(key_type = "Liferay Analytics Settings REST");
+		}
+
+		task ("And When clicking on the Save button") {
+			PortletEntry.save();
+		}
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase:114]: You can cp these two (https://github.com/liferay-frontend/liferay-portal/commit/1ccf831a0e46f89e779baf69086b6c6645555ab3 https://github.com/liferay-frontend/liferay-portal/commit/d2b24b84fd4918cb361e963bc2203ff200c7c7f2 )commits and do it this way:
```suggestion
		FrontendDataSetAdmin.createDataSet(
				key_name = "~!@#$%^&*(){}[],.<>/?",
				key_type = "Channel");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase [hunk 3]
```diff
@@ -92,4 +92,32 @@ definition {
...
+
+		task ("And Then Confirm that the dataset is created // Assert that name and provided as expected") {
+			AssertTextPresent(
+				locator1 = "FrontendDataSet#SELECT_CUSTOM_VIEW",
+				value1 = "~!@#$%^&*(){}[],.<>/?");
+		}
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase:120]:
```suggestion
			AssertTextPresent(
				locator1 = "FrontendDataSetAdmin#TABLE_CELL",
				key_itemName = "~!@#$%^&*(){}[],.<>/?");
		}
```

**Review by mateusralv — CHANGES_REQUESTED** [review]

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend

**Comment by andreluis2022** [comment]
@mateusralv

**Comment by andreluis2022** [comment]
ci:test:frontend

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend

**Review by mateusralv — APPROVED** [review]
Good Job @andreluis2022, after minor modifications, if `ci:test:sf` and `ci:test:frontend` pass you can forward Thank you 😄

**Comment by andreluis2022** [comment]
> Good Job @andreluis2022, after minor modifications, if `ci:test:sf` and `ci:test:frontend` pass you can forward Thank you smile

Ok thanks Mateus

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:forward

**Pull request closed without merging**